<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_NEW</class>
 <widget class="QWidget" name="MainWindow_NEW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1431</width>
    <height>786</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>1431</width>
     <height>721</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QWidget" name="tools_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>视图</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>48</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>　　画笔</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pro_pushbutton">
             <property name="text">
              <string>供区勾画</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="rec_pushbutton">
             <property name="text">
              <string>受区勾画画笔</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="rec_pushButton_dir">
             <property name="text">
              <string>受区发流方向</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="fjx_pushButton">
             <property name="text">
              <string>发际线画笔</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="sim_pushbuttion">
           <property name="text">
            <string>仿真效果</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="delete_pushButton">
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="clear_pushButton">
           <property name="text">
            <string>清空</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>绘制窗口</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QOpenGLWidget" name="openGLWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>仿真窗口</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QOpenGLWidget" name="openGLWidget_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>80</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1013</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="menu_plan">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>147</y>
      <width>136</width>
      <height>150</height>
     </rect>
    </property>
    <property name="title">
     <string>方案</string>
    </property>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
    <addaction name="action_8"/>
    <addaction name="action_9"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="action_10"/>
    <addaction name="action_11"/>
   </widget>
   <widget class="QMenu" name="menu_set">
    <property name="title">
     <string>设置</string>
    </property>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_plan"/>
   <addaction name="menu_edit"/>
   <addaction name="menu_set"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <action name="action">
   <property name="text">
    <string>新建档案</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>打开档案</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>导入</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>导出</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>新建方案</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>读取方案</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>生成报告</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>对比</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>档案编辑</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>设计参数</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>clear_pushButton</tabstop>
  <tabstop>pro_pushbutton</tabstop>
  <tabstop>rec_pushbutton</tabstop>
  <tabstop>rec_pushButton_dir</tabstop>
  <tabstop>sim_pushbuttion</tabstop>
  <tabstop>fjx_pushButton</tabstop>
  <tabstop>delete_pushButton</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
